float64[2] point
bool slow
---
bool success
string message
